WooCommerce/Classes/Authentication/Application Password/ApplicationPasswordAuthorizationWebViewController.swift

```diff
@@ -171,6 +189,15 @@ private extension ApplicationPasswordAuthorizationWebViewController {
         }
     }
 
+    /// Pops to the previous view controller (if provided) or pops one level otherwise.
+    @objc private func navigateToPreviousViewController() {
+        if let previous = previousViewController, let nav = navigationController {
```

Comment:
Super nit: we can simplify the `if let` checks now.

```suggestion
        if let previousViewController, let navigationController {
```